primap2/_convert.py

```diff
@@ -472,7 +449,6 @@ def derive_weights(
     dim: str,
```

Comment:
`derive_weights` function will change again when I remove `input_weights` and `output_weights`